lib/generated_plugin_registrant.dart

```diff
@@ -7,7 +7,6 @@
 
 import 'package:firebase_analytics_web/firebase_analytics_web.dart';
 import 'package:firebase_core_web/firebase_core_web.dart';
-import 'package:shared_preferences_web/shared_preferences_web.dart';
```

Comment:
Remove sharedPreferences because of rare usage despite of adding complexity to view generation code